projects/control-service/projects/model/apidefs/datajob-api/api.yaml

```diff
@@ -66,64 +66,6 @@ paths:
            in: path
            required: true
 
-  '/data-jobs/for-team/{team_name}':
```

Comment:
Does this need to be created manually? Or was it auto generated?

Reply:
What are you referring to ? the endpoint, or something else?

Reply:
I am asking does someone write in yaml manually or do we write the endpoint and then it auto populates this yaml?